cmake/dynamic_sourced_library.cmake

```diff
@@ -12,4 +12,4 @@ function(dynamic_sourced_cxx_library LIB_NAME WORK_DIR)
     set(lib_CPP_FILES_LIST ${lib_CPP_FILES})
 
     add_library(${LIB_NAME} STATIC ${lib_CPP_FILES_LIST})
-endfunction()
\ No newline at end of file
+endfunction()
```

Comment:
This file's change was something I noticed as I was implementing the main goal of this PR